tests/tests.schema

```diff
@@ -8143,6 +8143,12 @@
 						"https://mybot.contoso.com/api/skills/"
 					]
 				},
+				"connectionName": {
+					"$ref": "#/definitions/stringExpression",
+					"title": "OAuth Connection Name (SSO)",
+					"description": "The OAuth Connection Name, that would be used to perform Single SignOn with a skill.",
+					"default": "=settings.connectionName"
+				},
```

Comment:
Was the app.schema not regenerated from another check-in?  Seems like that would be hard unless you didn't run tests or didn't have bf installed, although it should help that bf-dialog is now part of core package.

Reply:
it means people are running unit test with out bf dialog:merge installed